assets/js/modules/adsense/components/common/UseSnippetSwitch.js

```diff
@@ -73,14 +73,10 @@ export default function UseSnippetSwitch( props ) {
 				/> <span className="googlesitekit-recommended">{ __( 'Recommended', 'google-site-kit' ) }</span>
 			</div>
 			{ useSnippet && checkedMessage &&
-				<SettingsNotice>
-					{ checkedMessage }
-				</SettingsNotice>
+				<SettingsNotice notice={ checkedMessage } />
```

Comment:
Let's fix formatting here. When we render conditional elements using multiline formatting, we should wrap them with parenthesis:

```javascript
{ A === B && (
    <SettingsNotice ... />
) }
```

We don't need parenthesis for the single approach:

```javascript
{ A === B && <SettingsNotice ... />  }
```

Also, when we have multiple conditions, they should be wrapped with parenthesis too:

```javascript
{ ( A === B && C ) && <SettingsNotice ... /> }
```